src/cljs/rems/spa.cljs

```diff
@@ -79,8 +79,7 @@
 (reg-sub
  :roles
  (fn [db _]
-   (when-let [roles (get-in db [:identity :roles])]
-     (set roles))))
```

Comment:
was this change intentional?

Reply:
Yes, see the commit. It's better to already originally store the set and not just use subscription because many places use the `:roles` from db directly.